langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -99,6 +117,21 @@ public isolated function get(xml<ItemType> x, int i) returns ItemType = @java:Me
 } external;
 
 # Concatenates xml and string values.
+# 
+# ```ballerina
+# xml bookA = xml `<book>Ballerina</book>`;
+# xml bookB = xml `<book>Java</book>`;
+# xml:concat(bookA, bookB, xml `<book>Python</book>`) ⇒ <book>Ballerina</book><book>Java</book><book>Python</book>
```

Comment:
Please refer to https://github.com/ballerina-platform/ballerina-lang/pull/38981#discussion_r1060286750

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/748ab266475d51db8f993ce05fca56be34858b08